src/FilterDetails/FilterDetails.tsx

```diff
@@ -7,6 +7,8 @@ import "./FilterDetails.scss";
 
 type Props = FilterDetailsProps & CallbackProps;
 
+const escKey = 27;
```

Comment:
Такие константы принято называть капсом и снейк-кейсом, чтобы сразу было понятно, что это прям Константа Константовна.
```suggestion
const ESC_KEY = 27;
```